### PR TITLE
Add docs for `HTMLIFrameElement.sandbox`

### PR DESCRIPTION
**Pull request by skyclouds2001**
### Description

spec at https://html.spec.whatwg.org/multipage/iframe-embed-object.html#attr-iframe-sandbox

### Related issues and pull requests

part of the https://github.com/mdn/mdn/issues/520

### CODE REVIEW

**Review by ShubhamOulkar — COMMENTED** [review]
repeating content is not necessary (value section and notes), just add reference links.

**Comment by skyclouds2001** [comment]
> repeating content is not necessary (value section and notes), just add reference links.

Good point!

File: files/en-us/web/api/htmliframeelement/sandbox/index.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+> **Note:** When redirecting the user, opening a popup window, or opening a new tab from an embedded page within an `<iframe>` with the `sandbox` attribute, the new browsing context is subject to the same `sandbox` restrictions. This can create issues — for example, if a page embedded within an `<iframe>` without a `sandbox="allow-forms"` or `sandbox="allow-popups-to-escape-sandbox"` attribute set on it opens a new site in a separate tab, form submission in that new browsing context will silently fail.
+
+> **Note:** This property only take effect when the content navigable of the `<iframe>` element is navigated. Removing them, or removing the entire `sandbox` attribute, has no effect on an already-loaded page.
+
```
Comment (skyclouds2001) [inline files/en-us/web/api/htmliframeelement/sandbox/index.md:20]:
```suggestion
```

**Review by teoli2003 — APPROVED** [review]
👍

**Pull request merged**